src/CryptoBank.Domain/Models/Token.cs

```diff
@@ -0,0 +1,9 @@
+namespace CryptoBank.Domain.Models;
+
+public record Token(string RefreshToken, int UserId, DateTimeOffset ExpirationTime)
```

Comment:
Yes, I agree, this name is too broad. But I could not use `RefreshToken` name, because it conflicted with `RefreshToken` method and request names. I could not find a better solution for this moment.